### PR TITLE
Remove mentions of IDL dictionaries

### PR DESCRIPTION
**Pull request by teoli2003**
Remove the mention of the names of the IDL dictionaries.

These names are not available in debuggers and doesn't bring anything to the reader.

### CODE REVIEW

File: files/en-us/web/api/clipboarditem/clipboarditem/index.md [hunk 1]
```diff
@@ -22,19 +22,18 @@ The **`ClipboardItem()`** constructor of the {{domxref("Clipboard API")}} create
...
   - : An {{jsxref("Object")}} with the {{Glossary("MIME type")}} as the key and data as the value. The data can be represented as a {{domxref("Blob")}}, a {{jsxref("String")}} or a {{jsxref("Promise")}} which resolves to either a blob or string.
-- `clipboardItemOptions` {{optional_inline}}
-
-  - : An {{jsxref("Object")}} with the following properties:
-
-    - **`presentationStyle`**: One of `"unspecified"`, `"inline"` or `"attachment"`. The default is `"unspecified"`.
+- `options` {{optional_inline}}
+  - : An object with the following properties:
+    - `presentationStyle` {{optional_inline}}
+      - : One of the 3 strings: `unspecified`, `inline` or `attachment`. The default is `unspecified`.
```
Comment (jpmedley) [inline files/en-us/web/api/clipboarditem/clipboarditem/index.md:36]:
```suggestion
      - : One of the three strings: `unspecified`, `inline` or `attachment`. The default is `unspecified`.
```

File: files/en-us/web/api/documenttimeline/documenttimeline/index.md [hunk 2]
```diff
@@ -25,7 +25,11 @@ new DocumentTimeline(options)
 ### Parameters
 
 - `options`
-  - : An object specifying options for the new timeline. Currently the only supported option is the `originTime` member which specifies the zero time for the `documentTimeline` as a real number of milliseconds relative to the {{domxref("PerformanceTiming.navigationStart","navigationStart")}} moment of the active document for the current browsing context.
+  - : An object specifying options for the new timeline. The following properties are available:
+    - `originTime`
+      - : A number which specifies the zero time for the `documentTimeline`
```
Comment (jpmedley) [inline files/en-us/web/api/documenttimeline/documenttimeline/index.md:30]:
```suggestion
      - : A number that specifies the zero time for the `documentTimeline`
```

File: files/en-us/web/api/documenttimeline/documenttimeline/index.md [hunk 3]
```diff
@@ -25,7 +25,11 @@ new DocumentTimeline(options)
...
+        as a real number of milliseconds relative to the {{domxref("PerformanceTiming.navigationStart","navigationStart")}} moment of the active document
```
Comment (jpmedley) [inline files/en-us/web/api/documenttimeline/documenttimeline/index.md:31]:
```suggestion
        as a real number of milliseconds relative to the {{domxref("PerformanceTiming.navigationStart","navigationStart")}} time of the active document
```

File: files/en-us/web/api/eventsource/eventsource/index.md [hunk 4]
```diff
@@ -19,20 +19,21 @@ remote resource.
...
-- `configuration` {{optional_inline}}
+- `options` {{optional_inline}}
 
   - : Provides options to configure the new connection. The possible entries are:
 
-    - `withCredentials`, defaulting to `false`, indicating if
-      CORS should be set to `include` credentials.
+    - `withCredentials` {{optional_inline}}
+      - : A boolean value, defaulting to `false`, indicating
+        if CORS should be set to `include` credentials.
```
Comment (jpmedley) [inline files/en-us/web/api/eventsource/eventsource/index.md:36]: This wording puzzles me. If 'include' is a verb, then it shouldn't have backticks. If its a reference to an `include` parameter, where is it described? Either way, I don't see how credentials are included.

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: files/en-us/web/api/eventsource/eventsource/index.md (hunk 4 above)
Comment (teoli2003) [reply files/en-us/web/api/eventsource/eventsource/index.md:36]: I think this is better [spec](https://html.spec.whatwg.org/multipage/server-sent-events.html#eventsourceinit)

```suggestion
        if the object's CORS credential mode should be set to `include`, or not.
```

Comment (OnkarRuikar) [reply files/en-us/web/api/eventsource/eventsource/index.md:36]: How about this?

> A boolean value that is `false` by default. Setting it to `true` sets the connection request's _credentials mode_ to "include". Request’s _credentials mode_ controls the flow of credentials during a fetch.

**Review by jpmedley — APPROVED** [review]

**Pull request merged**